core/errors.py

```diff
@@ -17,9 +21,19 @@ def validation_error_handler(error: ValidationError):
     }, 400
 
 
-def unimplemented_uc_error_handler():
-    return {
-        "detail": "Uncaught workbook validation failure",
-        "status": 500,
-        "title": "Bad Request",
-    }, 500
+def handle_exception(e: Exception):
+    # pass through HTTP errors
+    if isinstance(e, HTTPException):
```

Comment:
can we use a better var name than e? I know like every online example ever, for catching exceptions does this, but it'd be better as a proper var name I think